test/blackbox/common/DDSBlackboxTestsListeners.cpp

```diff
@@ -711,6 +712,42 @@ void sample_lost_test_dw_init(
 
                 return false;
             };
+    testTransport->drop_data_frag_messages_filter_ = [](eprosima::fastrtps::rtps::CDRMessage_t& msg)-> bool
+            {
+                uint32_t old_pos = msg.pos;
+
+                // see RTPS DDS 9.4.5.4 DataFrag Submessage
+                EntityId_t readerID, writerID;
```

Comment:
I'm okey with doing this with `skip-ci`
```suggestion
                EntityId_t readerID;
                EntityId_t writerID;
```

Reply:
6df3ea1